plane/source/docs/circle-mode.rst

```diff
@@ -6,7 +6,7 @@ CIRCLE Mode
 
 Circle mode is similar to :ref:`LOITER <loiter-mode>`, but doesn't attempt
 to hold position. This is primarily meant as a failsafe mode and is the
-mode that the aircraft will enter by default for 20 seconds when a
+mode that the aircraft will enter by default for ``FS_LONG_TIMEOUT`` seconds when a
```

Comment:
remove all following.

Reply:
Would it make sense to include a reference/link to the Failsafe doc page (and specifically the section that refers to actions here: https://ardupilot.org/plane/docs/apms-failsafe-function.html#failsafe-parameters-and-their-meanings)
How can I add that link?

Reply:
See :ref:`failsafe documentation <apms-failsafe-function>` for more information.

Reply:
the backtick does not appear in markdown comments like these...just highlites the phrase

Reply:
I don't know if I got it right - a backtick didn't show up in what you posted, so I just copied the text.

Reply:
fixed and squashed....merging